api/python/provisioner/commands/configure_setup.py

```diff
@@ -255,12 +172,32 @@ def run(self, path, number_of_nodes):  # noqa: C901
         node_list = []
         count = int(number_of_nodes)
 
+        pillar_map = {"srv_default": "cluster",
+                      "storage_enclosure_default": "storage_enclosure"}
+
         for section in content:
             input_type = section
             pillar_type = section
-            if 'srvnode' in section:
-                input_type = 'node'
-                pillar_type = f'cluster/{section}'
+            if 'default' in section:
+                #for section_name in ['srv_default', 'storage_enclosure_default']:
+                if 'srvnode' in section:
+                    input_type = 'srv_default'
```

Comment:
@divya-vijay  could you use some variable for `srv_default` and `storage_enclosure_default` (better -- constant like in upper case) here and in lines 175-177 (and in any other referenced places) to avoid hard coding? The same for `default` as an expected prefix, `cluster`, `storage_enclosure` and any other strings that make special meaning for our business logic. Thank you

Reply:
Addressed with CONSTANTS for all params in config file..